### DISCUSSION TITLE
HOWTO] waydroid apps half screen / portrait mode

### DISCUSSION THREAD

**Discussion by hoshi411** [waydroid/waydroid#928] [General]
Running waydroid on desktop/manjaro gnome here.

I don't like that the apps run full screen and are not resizable. Most apps would be better in a portrait mode rather than the desktop landscape. How do I force all the apps or any apps to open in portrait mode?

**Comment by Karamveer9200** [comment]
#Launch Waydroid In Multi-Window Mode:
#First we need to set the property while a Waydroid session is running:

 (in terminal) :       prop set persist.waydroid.multi_windows true                                    

#After that, we can restart the container:

 (in terminal)  :      systemctl restart waydroid-container                                                          

Then we are ready to launch an app, and it will start in multi-window mode   (android app icons will  also appear in gnome app  menu)

**Reply by Karamveer9200** [reply]
https://docs.waydro.id/usage/install-on-desktops#launch-waydroid-in-multi-window-mode

**Reply by Karamveer9200** [reply]
in multi window mode, you can very easily resize the application window to you specifications

**Comment by rexendevar** [comment]
I have the opposite problem. I switched to multi-window mode, but I want to be able to resize the windows so that they take up my full screen. How can I do this without switching back to single window mode?

**Reply by S7venLights** [reply]
I would guess `prop set persist.waydroid.multi_windows false`

But seems like apps remember the size you set for the window and there is a maximise button.

**Reply by akhil-flycatch** [reply]
this one worked for me
